demos/mkdocs/appyx-interactions/interactions/sample4/web/src/jsMain/kotlin/com.bumble.appyx.demos.sample4/main.js.kt

```diff
@@ -0,0 +1,45 @@
+package com.bumble.appyx.demos.sample4
```

Comment:
Let's have a distinct name for it. Sample1-2-3 are for the interactions front page (and can be argued they should have a better name too). All others already have their own name.

```suggestion
package com.bumble.appyx.demos.observemp
```